infrastructure/cloudconnect/README.md

```diff
@@ -0,0 +1,205 @@
+# CloudConnect - Establishing a VPN Connection Between AWS and GCP
+
+CloudConnect is a package that automates the setup of dual VPN tunnels between AWS and GCP. While this connection is **NOT** meant for high throughput, low latency connections (1Gbps+), it can certainly support basic management and configuration traffic. Currently it supports the creation of both _static-routes_ and _bgp_ connections.
+
+In [AWS](http://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/VPC_Route_Tables.html#route-tables-priority), route priority is defined as:
+
+- BGP propagated routes from an AWS Direct Connect connection
+- Manually added static routes for a VPN connection
+- BGP propagated routes from a VPN connection
+
+In [GCP](https://cloud.google.com/compute/docs/reference/latest/routes), it's  defined by the route table and priorities:
+
+- Routes match packets by destination IP address, preferring smaller or more specific ranges over larger ones.
+- If there is a tie, the system selects the route with the smallest priority value.
+- If there is still a tie, it uses the layer three and four packet headers to select just one of the remaining matching routes.
+
+A common practice is to have a _bgp_ enabled connection for network to network connectivity, and then enable _static routes_ between more bandwidth hungry subnets.
+
+
+## Prerequisites
+
+This document assumes one has _compute.networkAdmin_ rights in the Shared VPC host project, and AWS Credentials to run CloudFormation that manages a VPC. For the static routes, must have the list of AWS and GCP subnet cidr blocks.
```

Comment:
I'm pretty sure that we renamed XPN to Shared VPC, so I did a search and replace. Your right, that we can just say GCE to encompass everything...